frontend/packages/topology/src/entities/BaseGraphEntity.ts

```diff
@@ -48,6 +73,10 @@ export default class BaseGraphEntity<E extends Graph = Graph, D = any>
 
   setModel(model: E): void {
     super.setModel(model);
+
+    if ('layout' in model && model.layout) {
```

Comment:
Should be possible to nullify the layout.
```suggestion
    if ('layout' in model) {
```